app/src/main/java/com/github/se/signify/model/user/UserRepository.kt

```diff
@@ -89,4 +89,17 @@ interface UserRepository {
       onSuccess: () -> Unit,
       onFailure: (Exception) -> Unit
   )
+
+  fun getInitialQuestAccessDate(
+      userId: String,
+      onSuccess: (String?) -> Unit, // Date as "YYYY-MM-DD"
```

Comment:
Instead of having a one line comment, we could have a documentation for the arguments of the functions.